src/algorithms/cli.ts

```diff
@@ -13,44 +15,88 @@ const handleRejection: NodeJS.UnhandledRejectionListener = (err) => {
 
 process.on('unhandledRejection', handleRejection)
 
-function readJsonFromFile<T>(arg: number) {
-  const inputFilename = process.argv[arg]
-  if (!inputFilename) {
-    usage()
-  }
-  console.log(`Reading data from file ${inputFilename}`)
+/**
+ * Read a file in JSON format.
+ *
+ * @param inputFilename The path to the file.
+ */
+function readJsonFromFile<T>(inputFilename: string) {
+  console.log('Reading data from file ' + inputFilename)
   const inputData = fs.readFileSync(inputFilename, 'utf8')
   const inputJson = JSON.parse(inputData) as T
-  console.log(`Read input data`)
   return inputJson
 }
 
-async function main() {
-  const scenarioData = readJsonFromFile<ScenarioData>(2)
-  const outputFile = process.argv[3]
-  if (!outputFile) {
-    usage()
+/**
+ * Get severity distribution data. If a file is specified on the command
+ * line, give priority to its contents, else load a default distribution.
+ *
+ * @param inputFilename The path to the file.
+ */
+function getSeverity(inputFilename: string | undefined) {
+  if (inputFilename) {
+    const data = readJsonFromFile<any>(inputFilename)
+    return data.data
   }
-  console.log(`Will write model output to ${outputFile}`)
+  else {
+    const DEFAULT_SEVERITY_DISTRIBUTION = 'China CDC'
+    return getSeverityDistribution(DEFAULT_SEVERITY_DISTRIBUTION)
+  }
+}
 
-  const scenario = toInternal(scenarioData.data)
+/**
+ * Get age distribution data. If a file is specified on the command
+ * line, give priority to its contents, else load the distribution
+ * name as specified in the scenario parameters.
+ *
+ * @param inputFilename The path to the file.
+ * @param name The age distribution name to use if no file is
+ *             specified.
+ */
+function getAge(inputFilename: string | undefined, name: string) {
+  if (inputFilename) {
+    const data = readJsonFromFile<any>(inputFilename)
```

Comment:
I think you can type this as `AgeDistributionData` instead of `any`. That type can be added to the import from `./types/Param.types`.